docs/azmcp-commands.md

```diff
@@ -518,6 +518,15 @@ azmcp extension az --command "storage account show --name <account-name> --resou
 azmcp extension az --command "vm list --resource-group <resource-group>"
 ```
 
+### Azure Quick Review CLI Extension Operations
+```base
+# Scan a subscription for recommendations
+azmcp extension azqr --subscription-id "<subscriptionId>"
+
+# Scan a subscription and scope to a specific resource group
+azmcp extension azqr --subscription-id "<subscription-id>" --resource-group-name "<resource-group-name>"
```

Comment:
Use --resource-group

Reply:
Updated to use the global --subscription and --resource-group option.